lib/statum/event.rb

```diff
@@ -1,7 +1,7 @@
 module Statum
   # Class for storing event info
   class Event
-    attr_reader :from, :to, :before, :after
+    attr_reader :from, :to
```

Comment:
Would this changes be reverted in favor of #4 ?
Single responsobility principle suggest not to add logic-specific methods to model. 

It also makes things clearer, if proc application logic is not spread